### DISCUSSION TITLE
Is there a way to use a value from a ConfigMap inside HelmRelease values?

### DISCUSSION THREAD

**Discussion by fernandocamargoai** [fluxcd/flux2#1118] [Q&A] [answered]
I have a git syncing multiple clusters and each of them has different domain names. I have a HelmRelease for the prometheus-stack, which includes Grafana and I need to pass my domain for the Grafana. I know one option would be to use overlays to replace only the part that sets the domain name, but it would be much better to somehow use a value from a ConfigMap. This way, I'd have different instances of a given ConfigMap for each cluster and one of my variables would be the domain_name. Is there a way to do it, or is overlay my only option?

**Comment by stefanprodan** [comment, marked as answer]
Yes checkout the docs: https://toolkit.fluxcd.io/components/helm/helmreleases/#values-overrides

**Reply by fernandocamargoai** [reply]
Awesome. Thank you!
